### ISSUE TITLE
Quick Actions for Command Handler

### ISSUE THREAD

**Issue by FSG-Cat** [the-draupnir-project/Draupnir#330]
The current flow to more easily ban a user is to type !draupnir ban USER and have the bot present you with the following choices in the following order. What list to write to and what reason. This requires you to click thru multiple prompt and wait for multiple rounds of reactions to be delivered to you and the delays involved. 

This feature is indended to allow you to configure ahead of time so called Quick Actions. Where you have a preset combination of policy list and reason and presented with this first before anything else in your bot. 

Some policy lists like for example CME main are Mono reason users due to the spam keyword being auto redact in legacy mjolnir and draupnir (But we can in the future add workarounds to this problem but it wouldnt fix legacy problems). So for a list like CME a quick action that says CME and spam as reason would be a single click. This is especially helpful as on desktop you can just type the whole thing but on Mobile its much easier to use the command handler. 

And with Draupnir MPS not wanting to keep shortcodes around unless we absolutely have to this is a compromise that gives the speed of shortcodes for bans that you have pre configured. 

This attached image is and example of how this UX could look in Draupnir. 
![image](https://github.com/the-draupnir-project/Draupnir/assets/48515417/5b174caa-3b53-4f27-9ce9-ece4500b929e)

Ofc this is without the reactions but still shows an example.

**Comment by Gnuxie** [comment]
So the issue is not asking for any more than preconfigured "list-reason" pairs?

**Comment by FSG-Cat** [comment]
Yes the issue is essentially just in fancier words describing what is showed in the image. A list of as you say "list-reason" pairings. The image attached shows one. List is CME and reason is spam now ofc its a bit confusing that it includes a short code. This is based on a pre Shortcode removal command handler output.
